components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -571,6 +696,162 @@ public void addDefaultEmailTemplates(String tenantDomain) throws I18nEmailMgtExc
         }
     }
 
+    /**
+     * Add the default notification templates which matches the given notification channel to the respective tenants
+     * registry.
+     *
+     * @param notificationChannel Notification channel (Eg: SMS, EMAIL)
+     * @param tenantDomain Tenant domain
+     * @throws NotificationTemplateManagerException Error adding the default notification templates
+     */
+    @Override
+    public void addDefaultNotificationTemplates(String notificationChannel, String tenantDomain)
+            throws NotificationTemplateManagerException {
+
+        // Get the list of Default notification templates.
+        List<NotificationTemplate> notificationTemplates =
+                getDefaultNotificationTemplates(notificationChannel);
+        int numberOfAddedTemplates = 0;
+        try {
+            for (NotificationTemplate template : notificationTemplates) {
+                String displayName = template.getDisplayName();
+                String type = I18nEmailUtil.getNormalizedName(displayName);
+                String path = buildTemplateRootDirectoryPath(type, notificationChannel);
+
+            /*Check for existence of each category, since some template may have migrated from earlier version
```

Comment:
Use line comments

Reply:
This is a multiline comment. Therefore, multiline comments will be used.